src/core/api/providers/litellm.ts

```diff
@@ -197,8 +198,8 @@ export class LiteLlmHandler implements ApiHandler {
 
 		let temperature: number | undefined = this.options.liteLlmModelInfo?.temperature ?? 0
 
-		if (isOminiModel && reasoningOn) {
-			temperature = undefined // Thinking mode doesn't support temperature
+		if (isOminiModel || (isAnthropicModelId(modelId) && reasoningOn)) {
```

Comment:
The logic here changes the original condition.  Originally we went into this code branch if it was an OminiModel and thinking was on.  The new change will go into this code branch for OminiModel regardless of thinking.   Please correct the parenthesis.